pyccl/boltzmann.py

```diff
@@ -248,6 +248,9 @@ def get_isitgr_pk_lin(cosmo):
     cp.ISiTGR_muSigma = True
     cp.mu0 = cosmo['mu_0']
     cp.Sigma0 = cosmo['sigma_0']
+    cp.c1 = cosmo['c1_mg']
+    cp.c2 = cosmo['c2_mg']
+    cp.Lambda = cosmo['lambda_mg']
```

Comment:
When adding these parameters to isitgr, will the power spectrum already contain these modifications? In which case, should we not correct for them in the tracers? I.e. are these applied to the matter power spectrum used by CCL?

Reply:
ISiTGR matter power spectrum already has the effect of mu but we are not using the Weyl power spectrum so the effects of Sigma is needed in tracers.py.